src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User | null
```

Comment:
You shouldn't render this component at all if the user is null
```suggestion
  user: User
```